CMSgov/easi-app
# [EASI-1939] - Display prepare for GRT/GRB meeting help links

# EASI-1939

## Changes and Description

- Display "Prepare for GRB Meeting" and "Prepare for GRT Meeting" help links in IT Governance Task List and IT Governance Steps Overview



## How to test this change

- Test IT Governance request in both GRT and GRB meeting phases
  - When phase is inactive, link should be unstyled
  - When phase is active, link should be primary CTA button 
- Start new IT Governance request to see steps overview links



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
